api/src/main/java/ai/djl/ndarray/NDArray.java

```diff
@@ -4415,4 +4415,98 @@ default String toDebugString(int maxSize, int maxDepth, int maxRows, int maxColu
     /** {@inheritDoc} */
     @Override
     void close();
+
+    /**
+     * Returns the norm of this {@code NDArray}.
+     *
+     * <p>Examples
+     *
+     * <pre>
+     * jshell&gt; NDArray array = manager.create(new float[] {-3f, -4f});
+     * jshell&gt; array.norm();
+     * ND: () cpu() float32
+     * 5.
+     * jshell&gt; NDArray array = manager.create(new float[] {1f, 2f, 3f, 4f}, new Shape(2, 2));
+     * jshell&gt; array.norm();
+     * ND: () cpu() float32
+     * 5.4472
+     * </pre>
+     *
+     * @return the norm of this {@code NDArray}
+     */
+    NDArray norm();
```

Comment:
You can add a default here with keepDims=false. Then, there are fewer methods to implement on the other classes.

Reply:
Done